README.md

```diff
@@ -109,30 +110,30 @@ To clone a project, use the command:
 git clone --recursive https://github.com/Gedsh/InviZible
 ```
 
-To build **InviZible Pro** please use **Android Studio**.
+To build **InviZible Pro**, please use **Android Studio**.
 
 If you see something like this:
 _Illegal character in opaque part at index 2: C:\KStore\keystore.properties_
 
-Please comment lines of the settings.gradle file in the project root, as shown below:
+Please comment out this line in the settings.gradle file of the project root, as shown below:
 
 ```bash
 include ':tordnscrypt', ':filepicker'
 project(':filepicker').projectDir = new File('android-filepicker/filepicker')
-//Please comment line below if you are not the project owner
+//Please comment out the line below if you are not the project owner
 //project(':tordnscrypt').buildFileName = 'owner.gradle'
 ```
 
-It is expected that you can solve build problems yourself, so there is no support for building. 
-If you cannot build yourself, there are prebuilt versions of **InviZible Pro** available [here](https://github.com/Gedsh/InviZible/releases/latest).
+As is expected, if you can solve how to build problems for yourself, so there is no support for how you got there. :) 
```

Comment:
What is it?

Reply:
Same joke I thought you were going for.

Reply:
Not funny, as for me. I often suggest that beginners read my github.

Reply:
Will change.

Reply:
```suggestion
There is no support offered for building the app yourself, as this is an advanced option.
```